cmd/rook/ceph/operator.go

```diff
@@ -91,14 +111,132 @@ func startOperator(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		rook.TerminateFatal(err)
 	}
-
 	rookImage := rook.GetOperatorImage(context.Clientset, containerName)
 	serviceAccountName := rook.GetOperatorServiceAccount(context.Clientset)
 	op := operator.New(context, volumeAttachment, rookImage, serviceAccountName)
+	if isPresent, err := isSecretPresent(context); isPresent && err == nil {
+		err = initWebhook(context)
+		if err != nil {
+			logger.Errorf("an error has occurred while initializing webhook: %+v", err)
```

Comment:
Errors should be separated by periods `.`, not colons `:`
```suggestion
			logger.Errorf("an error has occurred while initializing webhook. %+v", err)
```

Reply:
Rook is using `failed to ...` when reporting error so let's stick with that.

Reply:
Also do not print the full struct, use `%v`

Reply:
Fixed @BlaineEXE @leseb